io/io/src/TBufferMerger.cxx

```diff
@@ -69,7 +68,7 @@ void TBufferMerger::WriteOutputFile()
       merger.OutputFile(fName.c_str(), fOption.c_str(), fCompress);
    }
 
-   while (!done) {
+   for (;;) {
```

Comment:
It's a matter of style, but compilers may issue a warning for `while(true)` ("constant expression used as loop expression") but usually not for `for(;;)`.